config_file/MacOS.conf

```diff
@@ -0,0 +1,13 @@
+server {
+
+listen [::]:8080 default_server;
+
+server_name www.example.com;
+
+root ./root_contents/;
+
+index index.html;
+
+try_files $uri /index.html;
+
```

Comment:
Thank you for your work! There was a lot of things I didn't mention in my flowchart, but you made this part very well :)
I want to add some elements to the config file, but I think that it's better to merge this commit once. After that, I will add another config file.